src/styles/variables/_theme.scss

```diff
@@ -88,6 +88,7 @@ $fold: (
 );
 // Buttons
 $buttons: (
+  menu:(padding-top: 10px),
```

Comment:
hmm I would perhaps not do this, and instead use one of the spacing variables we have eg, `padding-top: var(--spacing-quarter-unit);` on the `.button--menu` class, so that we can keep all spacing units consistent